### PR TITLE
COG-1388 create tiles on search page

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to (link)?
-Subtask to show tiles on the search page(landing page of manage users).
https://osi-cwds.atlassian.net/browse/COG-1372

#### Brief feature description
- show tiles on the search page(landing page of manage users).
- click events will be added in the next subtask.

### CODE REVIEW

**Comment by sunilb0575** [comment]
retest this please.

File: app/javascript/actions/searchUserListActions.js [hunk 1]
```diff
@@ -39,6 +39,17 @@ export const setSearch = query => ({
   payload: query,
 })
 
+/**
+ * Set search for tiles
+ * @param {Object[]} query
+ * @param {string} query[].field Identifier for the field on which to search
+ * @param {string|number|boolean} query[].value Value on which to search
```
Comment (kelfink) [inline app/javascript/actions/searchUserListActions.js]: how do these params correlate to the method?  I don't follow it

File: app/javascript/reducers/searchUserListReducers.js [hunk 2]
```diff
@@ -56,6 +62,66 @@ function searchUserListReducer(state = initialValue, { type, payload, error, met
         users: null,
       }
 
+    case actionTypes.GET_ACTIVE_USERS_REQUEST:
+      return { ...state, fetching: true, error: null, query: payload.query }
+
+    case actionTypes.GET_ACTIVE_USERS_SUCCESS:
+      state.searchPageTiles[0].count = payload.meta.total
```
Comment (kelfink) [inline app/javascript/reducers/searchUserListReducers.js]: Named indices would be safer than magic numbers.  What if we decide to add a tile in the first position and move the others down?  We'd have to renumber everything and chase the values around.

Comment (kelfink) [reply app/javascript/reducers/searchUserListReducers.js]: This sets the count of users based on the ES response's meta's total.  Does this mean all the 100's of users are retrieved, passed back to the front end, then discarded so we only use the count ?  That seems inefficient, if so.

**Comment by sunilb0575** [comment]
@kelfink Seems like this is the best way to update a value inside an array of objects.

File: app/javascript/actions/searchUserListActions.js (hunk 1 above)
Comment (sunilb0575) [reply app/javascript/actions/searchUserListActions.js]: Here the params are type and query.

File: app/javascript/reducers/searchUserListReducers.js (hunk 2 above)
Comment (sunilb0575) [reply app/javascript/reducers/searchUserListReducers.js]: Agree, changed to named indices.
To make it efficient, passing the resulting users list to the filter list page.

File: app/javascript/actions/searchUserListActions.js (hunk 1 above)
Comment (kelfink) [reply app/javascript/actions/searchUserListActions.js]: Ok, so it's describing to me that query needs to be of the form {field: "identifier", value: string|number|boolean}

**Pull request merged**